client/blocks/login/social.jsx

```diff
@@ -114,8 +114,39 @@ class SocialLoginForm extends Component {
 		);
 	};
 
-	// eslint-disable-next-line no-unused-vars
-	handleGitHubResponse = ( response ) => {};
+	handleGitHubResponse = ( access_token, triggeredByUser = true ) => {
+		const { onSuccess, socialService } = this.props;
+		let redirectTo = this.props.redirectTo;
+
+		// ignore response if the user did not click on the GitHub button
+		// and did not follow the redirect flow
+		if ( ! triggeredByUser && socialService !== 'github' ) {
+			return;
+		}
+
+		// load persisted redirect_to url from session storage, needed for redirect_to to work with GitHub redirect flow
+		if ( ! triggeredByUser && ! redirectTo ) {
+			redirectTo = window.sessionStorage.getItem( 'login_redirect_to' );
+		}
```

Comment:
I might be missing something, but I don't think this section is needed for our case - since `triggeredByUser` should be always `true` when we call `handleGitHubResponse` with `access_token` parameter only . 🤔